src/config.json

```diff
@@ -682,16 +682,14 @@
     "server": {
       "url": "http://localhost:9999/arlas",
       "collection": {
-        "name": "demo_ais-danmark",
+        "name": "demo_birdtracking-movebank",
```

Comment:
the widget are based on ais collection fields no ?

Reply:
Done